doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -245,41 +241,12 @@ Bluetooth Fast Pair samples
 Bluetooth Mesh samples
 ----------------------
 
-* Added:
```

Comment:
Entries of Bluetooth Fast Pair samples could also be removed - (line no: 237 to 239).